idl/chimera.pro

```diff
@@ -124,16 +135,19 @@ formtab[0]='ID      XCEN       YCEN   CENTROID       X_EB       Y_EB       X_WB
 formtab[1]='num        "          "         H°          "          "          "          "          "          "          "          "         H°          °       Mm^2          %          G          G          G          G          G          G          G         Mx         Mx         Mx'
 
 ;=====Sort data by wavelength=====
-reord=sort(ind.wavelnth)
-ind[*]=ind[reord]
-data[*,*,*]=data[*,*,reord]
+;reord=sort(ind.wavelnth)
+;ind[*]=ind[reord]
+;data[*,*,*]=data[*,*,reord]
 
 
 ;=====Normalises data with respect to exposure time=====
 for i=0,2 do data[*,*,i]=data[*,*,i]/ind[i].exptime
 
 ;=====removes negative data values=====
-data[where(data lt 0)]=0
+www=where(data lt 0,count)
+if count gt 0 then begin
+	data[www]=0
+endif
```

Comment:
Good one! because if you didn't get any... IDL8 would silently only change the last element of `data`!!
By the way, what does `www` mean? I would call it `label_neg`